sign in/style.css

```diff
@@ -0,0 +1,109 @@
+body
+{
+    margin: 0;
+    padding: 0;
+    background: url(banner.jpg);
+    background-size: cover;
+    background-position: center;
+    font-family: sans-serif; 
+}
+.login_box
+{
+    width: 345px;
+    background: rgb(0,0,0,0.5);
+    color: #fff;
+    top: 50%;
+    left: 50%;
+    position: absolute;
+    transform: translate(-50%,-50%);
+    box-sizing: border-box;
+    padding: 70px 20px;
+}
+h1
+{
+    margin: 0;
+    padding: 15 0 15px;
+    text-align: center;
+    font-size: 30px;
+    font-family:cursive;
+    
+}
+.login_box p
+{
+    margin: 0;
+    padding: 0;
+    font-weight: bold;
+    
+}
+.login_box input
+{
+    width: 100%;
+    padding-left: 30px;
+    transition: all 0.3s ease;
+    
+    
+    
+}
+form .input_box{
+  height: 60px;
+    width: 100%;
+    margin-top : 30px;
+    position: relative;
+}
+
+.login_box input[type="text"],[type="password"]
+{
+    border: none;
+    border-bottom: 1px solid #fff;
+    background: transparent;
+    height: 35px;
+    color:#fff;
+    font-size: 15px;
+    
+    
+}
+.input_box .icons
+{
+    position: absolute;
+    top: 50%;
+    left: 3%;
+    color:orange;
+    transform: translateY(-130%);
+    
+}
+.login_box input[type="submit"]
+{
+    border: none;
+    outline: none;
+    height: 45px;
+    background-color:darkorange;
+    color:#fff;
+    font-size: 19px;
+    border-radius: 20px;
+    padding-left: 8px;
+    
+}
+.login_box input[type="submit"]:hover
+{
+    cursor: pointer;
+    background: orangered;
+}
+
+.sign_up a
+{
+    color: #fff;
+    font-size: 14px;
+     text-decoration: none;
+    margin-top: 30px;
+    margin-left: 50px;
+
+}
+img{
+    width: 75px;
+    margin-top: -110px;
+    margin-left: 113px;
+}
+
+
```

Comment:
### One new line is enough
Line 106 is enough as empty line at the end of the file.
Lines 107, 108 and 109 is not necessary.